app/models/manageiq/providers/ibm_cloud/power_virtual_servers/cloud_manager/vm/operations.rb

```diff
@@ -16,6 +16,19 @@ def raw_create_snapshot(name, desc = nil, _memory = nil)
     raise MiqException::MiqVmSnapshotError, err.to_s
   end
 
+  def raw_resize(options)
+    with_provider_connection(:service => 'PCloudPVMInstancesApi') do |api|
+      body = IbmCloudPower::PVMInstanceUpdate.new("memory"     => options["memory"].to_i,
+                                                  "processors" => options["cores"].to_f,
+                                                  "proc_type"  => options["coreType"],
+                                                  "pin_policy" => options["pinning"])
```

Comment:
Any reason not to name the DDF option the same as the value that `PVMInstanceUpdate` is expecting?  Why name [[this]](https://github.com/ManageIQ/manageiq-providers-ibm_cloud/pull/423/files#diff-ee71765cda793a11b1ceeedbb4530ecefd066ddd46c797f14c75a0875f1ac60aR171) `coreType` instead of `proc_type`

Reply:
I had named it coreType as in the UI the label is 'Core Type', but I can change the name and id to the value that PVMInstanceUpdate expects if you think that makes more sense?

Reply:
Yeah no one is ever going to see the internal name/id (that is what label is for) so we can name it anything we want